runtime/interpreter/value.go

```diff
@@ -7354,6 +7432,8 @@ func NewDeferredDictionaryValue(
 	version uint16,
 ) *DictionaryValue {
 	return &DictionaryValue{
+		// TODO: type
```

Comment:
Looks like we missed a TODO here for dictionaries too. Could you please resolve it?